### ISSUE TITLE
Keyboard geometry doesn't change on rotation on Android 13

### ISSUE THREAD

**Issue by petershh** [dslul/openboard#781]
**Describe the bug**
In landscape mode, the keyboard appears like in portrait mode.

**To Reproduce**
1. Invoke the keyboard while holding your device vertically
2. Rotate your device

Or just invoke a keyboard while in landscape mode.

**Expected behavior**
Keyboard becomes lower and wider to accomidate landscape.

**Screenshots**
![Screenshot_20230409_025620](https://user-images.githubusercontent.com/23362279/230747915-a76baaa0-025b-4500-b3a9-bede0c7ddb43.jpg)

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy S20 FE
 - OS: Android 13

**Additional context**
- After changing theme, bug disappears, but switching to different keyboard and back to Openboard brings the issue back.
- Auto Rotate is on.

**Comment by RHJihan** [comment]
I also faced this issue.
Samsung Galaxy M32
Android 13 (One UI 5.1)

**Comment by Mrnofish** [comment]
I'm not getting a turn indicator on A13 when the keyboard is open, regardless of the keyboard. I honestly don't remember whether this was the case on previous Android versions, as I almost never use landscape mode.

Are you using Auto Rotate or some such by any chance ?

> Or just invoke a keyboard while in landscape mode.

I cannot reproduce this on LOS 20, perhaps it is specific to Samsung.

**Comment by petershh** [comment]
Yes, I am using Auto Rotate.

**Comment by Mrnofish** [comment]
Might want to edit the ticket to add that information.

FWIW I can reproduce with Auto Rotate, but not when opening the keyboard in landscape mode.

**Comment by Ammako** [comment]
It works here with auto-rotate, so it may be something else Samsung does.

**Comment by Blend0963** [comment]
I have a Google Pixel 6 and i have also this issue. So it's not specific to Samsung.

**Comment by mohzart** [comment]
same here... 
os:A12 (iode 3.7)
device: S10

p.s. guess i'm back to AnySoft...

so somehow it might be only on samsung devices
![Screenshot_20230625-151015_QKSMS](https://github.com/openboard-team/openboard/assets/73794389/a50b29a8-50ba-4bae-b080-2b1f43730093)

**Comment by RHJihan** [comment]
@Helium314, would you please take a look into this?

**Comment by Mrnofish** [comment]
@Helium314 you had me at multi-lingual typing.

Going to install your debug version, where should we be reporting bugs ?

**Comment by Helium314** [comment]
> @Helium314, would you please take a look into this?

Sorry, I'm on Android 9 and can't reproduce the issue there...

> where should we be reporting bugs ?

I enabled issues in the repo

**Comment by Incomptus** [comment]
Same issue here vivo y72 5g A13

**Comment by vertigo220** [comment]
I haven't experienced this, but I have a similar issue, also Samsung (A71 5G UW), Android 13. After the phone is in landscape mode, when switching to portrait, sometimes the keyboard doesn't seem to switch back, so it stays the same size as landscape, which means only roughly the left half fits on screen. This has only happened after being in the car and seems to possibly be related to a car mode functionality, but I've disabled that and it still happened. If I switch to another keyboard, that will be fine, and when I switch back, it's still messed up. It just eventually fixes itself.

![OpenBoard Issue](https://github.com/openboard-team/openboard/assets/12024557/707f21b0-176e-49d2-912f-cc86e519e075)

**Comment by vertigo220** [comment]
And right after I said I hadn't experienced this issue, I did today for the first time since I started using Openboard a couple years or so ago. So I've had it happen both ways: landscape mode and cut off in portrait mode and portrait mode using only half the screen width in landscape mode.

**Comment by Drubaloo** [comment]
2 years later. This bug is still present and predominant.

**Comment by mohzart** [comment]
Yup, and that's why i switched to "FrorisBoard" (on F-Droid; more traditional) and then into "Unexcpected Keyboard" (also on F-Droid; steep learning curve [special letters/signs/numbers have to be swiped instead of pressed longer] ...but when you learn it - you'll never go back)

**Comment by Drubaloo** [comment]
Excellent. I was just shopping for an alternative. I'll look into them

**Comment by DJCrashdummy** [comment]
if you haven't noticed, OpenBoard seems to not be maintained since ~3 years.

...but there is an active fork [HeliBoard](https://github.com/Helium314/HeliBoard) which was linked right above your first message, @Drubaloo.
BTW: this issue seems to be solved there: https://github.com/Helium314/HeliBoard/issues/401
